extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-spark-history-chart/values.yaml

```diff
@@ -7,9 +7,9 @@ deployment:
   labels: {}
   replicas: 1
   image:
-    repository: "apache/spark"
+    repository: "ghcr.io/boozallen/aissemble-spark"
     imagePullPolicy: IfNotPresent
-    tag: "3.5.1"
+    tag: "1.10.0-SNAPSHOT"
```

Comment:
i: Follow on to make this version dynamic. Probably through the typical values template or updating the plugin